pkg/proxy/apis/config/scheme/scheme.go

```diff
@@ -29,7 +29,7 @@ var (
 	Scheme = runtime.NewScheme()
 	// Codecs provides methods for retrieving codecs and serializers for specific
 	// versions and content types.
-	Codecs = serializer.NewCodecFactory(Scheme)
+	Codecs = serializer.NewCodecFactory(Scheme, serializer.EnableStrict)
```

Comment:
@obitech I noticed this merged, did you plan to add a lenient path to this PR before then?